app/src/main/java/in/ac/iitb/gymkhana/iitbapp/fragments/MapFragment.java

```diff
@@ -34,6 +37,20 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
     @Override
     public void onMapReady(GoogleMap googleMap) {
 
+        if (ContextCompat.checkSelfPermission(getActivity(), android.Manifest.permission.ACCESS_FINE_LOCATION) ==
+                PackageManager.PERMISSION_GRANTED &&
+                ContextCompat.checkSelfPermission(getActivity(), android.Manifest.permission.ACCESS_COARSE_LOCATION) ==
+                        PackageManager.PERMISSION_GRANTED){
+            googleMap.setMyLocationEnabled(true);
+            googleMap.getUiSettings().setMyLocationButtonEnabled(true);
+
+        }
+
+
+        LatLngBounds iitbBounds=new LatLngBounds(new LatLng(19.1249000, 72.9046000),new LatLng(19.143522, 72.920000));
+        googleMap.setLatLngBoundsForCameraTarget(iitbBounds);
+        googleMap.setMaxZoomPreference(30);
+        googleMap.setMinZoomPreference(18);
```

Comment:
If u remove min zoom bound, the map wont get restricted to the iitb area... Just that u cannot zoom in outside the area in that case